resolver-dns/src/main/java/io/netty/resolver/dns/DnsNameResolver.java

```diff
@@ -373,28 +410,57 @@ private static void setSuccess(Promise<InetAddress> promise, InetAddress result)
         }
     }
 
+    boolean searchDomains(String hostname) {
```

Comment:
can you clarify this method name and/or add javadoc? does this test if `hostName` is a search domain?